src/charm.py

```diff
@@ -768,8 +768,17 @@ def _on_config_changed(self, _):
                     "Duplicate route found; cannot add ingress. Run juju debug-log for details."
                 )
                 return
+        self.unit.set_workload_version(self._get_kubernetes_library_version())
```

Comment:
I wonder what the trade-offs would be for putting `kubernetes.__version__` here instead of creating another function?

Reply:
The story had a reference to Indico and this is how was done there as you can see [here](https://github.com/canonical/indico-operator/blob/main/src/charm.py#L262). I thought would be nice to follow the same pattern. Also, if by any chance would be needed to format the version string, it would be easier.